### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [OTP-react-redux/OTP-RR Main UI](https://hosted.weblate.org/projects/otp-react-redux/otp-rr-main-ui/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: i18n/zh.yml [hunk 1]
```diff
@@ -274,7 +274,6 @@ components:
     incoming_at: 接近 {stop}
     stopped_at: 巴士车门在 {stop} 打开
     travelingAt: 以 {milesPerHour}
-  vehicleName: '车辆 {vehicleNumber}'
```
Comment (AdrianaCeric) [inline i18n/zh.yml:277]: interesting ... this should not be deleted, rather indented

File: i18n/vi.yml [hunk 2]
```diff
@@ -164,7 +164,6 @@ components:
   PlaceEditor:
     genericLocationPlaceholder: Tìm kiếm vị trí
     locationPlaceholder: Tìm kiếm vị trí {placeName}
-    namePlaceholder: Đặt tên địa điểm
```
Comment (AdrianaCeric) [inline i18n/vi.yml:167]: Does anyone know why this is going to be deleted?

Comment (binh-dam-ibigroup) [reply i18n/vi.yml:167]: This string was deleted from the English file in #782.

File: i18n/zh.yml (hunk 1 above)
Comment (binh-dam-ibigroup) [reply i18n/zh.yml:277]: Correct. See updated change in line below (I reverted the delete in Weblate).

Comment (AdrianaCeric) [reply i18n/zh.yml:277]: Thank you!

**Review by AdrianaCeric — APPROVED** [review]

**Review by binh-dam-ibigroup — APPROVED** [review]

**Pull request merged**